app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt

```diff
@@ -94,7 +94,10 @@ class TokenHistoryPresenter(
             if (isRefresh) {
                 transactions.clear()
             }
-            val fetchedItems = historyInteractor.loadTransactions(token.publicKey, isRefresh)
+            val fetchedItems = historyInteractor.loadTransactions(
+                token.publicKey,
+                isRefresh = isRefresh || transactions.isEmpty()
```

Comment:
I am not sure if it's a best solution. 
cannot get how this field effects the empty state

Reply:
yes i agree
i'm consulting with @davran1001 
but it looks like issue in caches of states
and reloading fixes it

i will spend more time to check the root cause of this issue

Reply:
then it's wip?

Reply:
fixed @1sakov